### DISCUSSION TITLE
Config vertcoin'json miningcore

### DISCUSSION THREAD

**Discussion by Miguel974** [community/community#108379] [Programming Help]
### Body

Hello and is it possible to have help to configure the vertcoin vtc on miningcore config.json correctly

### Guidelines

- [X] I have read and understood this category's [guidelines](https://github.com/orgs/community/discussions/51384) before making this post.

**Comment by kyuna0312** [comment]
okey   then close this xD.

**Reply by Miguel974** [reply]
!

**Comment by Lanrad23** [comment]
"As I recall, configuring VTC on MiningCore requires setting up the config.json file with the correct parameters for your mining pool.
But there are a couple of important points to adjust:
""address"": Replace YourVertcoinWalletAddressHere with your actual Vertcoin wallet address.
""daemons"": The Vertcoin daemon (vertcoind) connection details (host, port, user, password) must match your vertcoind configuration.
""ports"": Configure mining ports based on the type of hardware miners will use to connect to your pool.
""paymentProcessing"": Adjust minimumPayment and other settings according to how and when you want miners to be paid out.
After configuring config.json, restart MiningCore for the changes to take effect. Always ensure your Vertcoin daemon is up-to-date and properly synchronized with the Vertcoin network."

**Reply by Miguel974** [reply]
Thank you for your response
And how well fixed this part

blockRefreshInterval": 120,
        "clientConnectionTimeout": 600,
        "banning": {
          "enabled": true,
          "time": 600,
          "invalidPercent": 50,
          "checkThreshold": 50
        },
        "ports": {
          "4073": {
            "name": "GPU-SMALL",
            "listenAddress": "*",
            "difficulty": 0.1,
            "varDiff": {
              "minDiff": 0.1,
              "maxDiff": null,
              "targetTime": 15,
              "retargetTime": 90,
              "variancePercent": 30

**Reply by Miguel974** [reply]
How well set it's parameters

**Reply by lurchinms** [reply]
di you ever get vertcoin to work?
